pac/Jenkins-CAPIF-build.groovy

```diff
@@ -0,0 +1,102 @@
+                    '''
+                }
+            }
+        }
+        stage('Get the code!') {
+            steps {
+                dir ("${env.WORKSPACE}/") {
+                    sh '''
+                    rm -rf $NETAPP_NAME 
+                    mkdir $NETAPP_NAME 
+                    cd $NETAPP_NAME
+                    git clone --single-branch --branch $GIT_CAPIF_BRANCH $GIT_CAPIF_URL .
+                    '''
+                }
+           }
+        }
+        stage('Build CAPIF') { 
+            steps {
+                dir ("${env.WORKSPACE}/${NETAPP_NAME}/services") {
+                    sh '''
+                    docker-compose up --build --force-recreate -d
+                    '''
+                }
+            }
+        }
+        stage('Modify image name and publish in AWS') { 
+            steps {
+                withCredentials([[$class: 'AmazonWebServicesCredentialsBinding', credentialsId: 'evolved5g-push', accessKeyVariable: 'AWS_ACCESS_KEY_ID', secretKeyVariable: 'AWS_SECRET_ACCESS_KEY']]) {               
+                    script {    
+                        def cmd = "docker ps --format '{{.Image}}'"
+                        def cmd2 = "docker ps --format '{{.Names}}'"
+                        def image = sh(returnStdout: true, script: cmd).trim()
+                        def name  = sh(returnStdout: true, script: cmd2).trim()
+                        sh '''$(aws ecr get-login --no-include-email)'''
+                        [image.tokenize(), name.tokenize()].transpose().each { x ->
+                            sh """ docker tag ${x[0]} ${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_DEFAULT_REGION}.amazonaws.com/evolved5g:${NETAPP_NAME}-${x[1]}-${VERSION}"""
+                            sh """ docker tag ${x[0]} ${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_DEFAULT_REGION}.amazonaws.com/evolved5g:${NETAPP_NAME}-${x[1]}-latest"""
+                            sh """ docker image push ${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_DEFAULT_REGION}.amazonaws.com/evolved5g:${NETAPP_NAME}-${x[1]}-${VERSION}"""
+                            sh """ docker image push ${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_DEFAULT_REGION}.amazonaws.com/evolved5g:${NETAPP_NAME}-${x[1]}-latest"""
+                        }
+                    }
+                }
+            }
+        }  
+        stage('Publish in Artifacotry') {
+            steps {
+                withCredentials([usernamePassword(credentialsId: 'docker_pull_cred', usernameVariable: 'ARTIFACTORY_USER', passwordVariable: 'ARTIFACTORY_CREDENTIALS')]) {
+                    dir ("${env.WORKSPACE}/${NETAPP_NAME}/services") {
+                        sh '''
+                            docker login --username ${ARTIFACTORY_USER} --password "${ARTIFACTORY_CREDENTIALS}" dockerhub.hi.inet
+                            docker-compose push
+                        '''
+                    }
+                }
+            }
+        }
+        stage('Cleaning docker images and containers') {
```

Comment:
maybe the cleaning of this pipeline should be added to a always section in this groovy. Just to be sure after any result of it, workspace is correctly cleaned.